sixMans/sixMans.py

```diff
@@ -681,234 +763,143 @@ async def setDefaultTeamSelection(self, ctx, team_selection_method):
         await ctx.send("Six Mans team selection is currently set to **{}**.".format(team_selection))
 
     @commands.guild_only()
     @commands.command()
     @checks.admin_or_permissions(manage_guild=True)
-    async def setCategory(self, ctx, category_channel: discord.CategoryChannel):
-        """Sets the 6 mans category channel where all 6 mans channels will be created under"""
-        await self._save_category(ctx, category_channel.id)
+    async def setCategory(self, ctx: Context, category_channel: discord.CategoryChannel):
+        """Sets the category channel where all game channels will be created under"""
+        await self._save_category(ctx.guild, category_channel.id)
         await ctx.send("Done")
 
     @commands.guild_only()
     @commands.command()
     @checks.admin_or_permissions(manage_guild=True)
-    async def getCategory(self, ctx):
+    async def getCategory(self, ctx: Context):
         """Gets the channel currently assigned as the transaction channel"""
         try:
-            await ctx.send("6 mans category channel set to: {0}".format((await self._category(ctx)).mention))
+            await ctx.send("{0} Mans category channel set to: {1}".format(self.teamSize, (await self._category(ctx.guild)).mention))
         except:
-            await ctx.send(":x: 6 mans category channel not set")
+            await ctx.send(":x: {0} Mans category channel not set".format(self.teamSize))
 
     @commands.guild_only()
     @commands.command()
     @checks.admin_or_permissions(manage_guild=True)
-    async def unsetCategory(self, ctx):
-        """Unsets the 6 mans category channel. 6 mans channels will not be created if this is not set"""
-        category = await self._category(ctx)
+    async def unsetCategory(self, ctx: Context):
+        """Unsets the category channel. Game channels will not be created if this is not set"""
+        category = await self._category(ctx.guild)
         old_helper_role = await self._helper_role(ctx.guild)
         if old_helper_role and category:
             await category.set_permissions(old_helper_role, overwrite=None)
-        await self._save_category(ctx, None)
+        await self._save_category(ctx.guild, None)
         await ctx.send("Done")
 
     @commands.guild_only()
     @commands.command()
     @checks.admin_or_permissions(manage_guild=True)
-    async def setHelperRole(self, ctx, helper_role: discord.Role):
-        """Sets the 6 mans helper role. Anyone with this role will be able to see all the game channels that are created"""
-        await self._save_helper_role(ctx, helper_role.id)
-        category = await self._category(ctx)
-        # await category.edit(overwrites={helper_role: discord.PermissionOverwrite(read_messages=True, manage_channels=True, connect=True)})
+    async def setHelperRole(self, ctx: Context, helper_role: discord.Role):
+        """Sets the 6 Mans helper role. Anyone with this role will be able to see all the game channels that are created"""
```

Comment:
We talked about this in discord but I wanted to note here that we ended up deciding to leave as is because we couldn't find a good way to format the help string without relying on constant variables which defeats the purpose of making the queue size differ between guilds